base/subarray.jl

```diff
@@ -527,7 +527,22 @@ function parentdims(s::SubArray)
 end
 
 ## Scalar indexing
-getindex(V::SubArray, I::Int...) = (@_inline_meta; checkbounds(V, I...); unsafe_getindex(V, I...))
```

Comment:
We could presumably use a loop `for func in (:getindex, :unsafe_getindex)`? But I'm fine either way.